timesketch/frontend-ng/src/components/Scenarios/Question.vue

```diff
@@ -87,14 +96,17 @@ export default {
     return {
       expanded: false,
       fullDescription: false,
+      conclusionText: '',
     }
   },
   computed: {
     sketch() {
       return this.$store.state.sketch
     },
   },
-  methods: {},
+  methods: {
+    saveConclusion: function () {},
+  },
```

Comment:
Is this working as intended that the ´saveConclusion´ function does not do anything, yet?

Reply:
Oh, good catch. That was a placeholder for my upcoming PR.